### PR TITLE
#1564 Add nrt-clean function.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: util/envDynawo.sh [hunk 1]
```diff
@@ -1372,6 +1373,15 @@ nrt() {
   fi
 }
 
+nrt_clean() {
+  find $DYNAWO_NRT_DIR/data -depth -type d -name "outputs*" -not -path "*reference*/*" -exec rm -rf {} \; > /dev/null 2>&1
+  rm -rf $DYNAWO_NRT_DIR/output
+  rm -rf $DYNAWO_HOME/util/nrt_diff/output
+  find $DYNAWO_NRT_DIR -name "*.pyc" -exec rm -rf {} \;
+  find $DYNAWO_HOME/util/nrt_diff -name "*.pyc" -exec rm -rf {} \;
+  find $DYNAWO_NRT_DIR/data -empty -type d -delete
```
Comment (rosiereflo) [inline util/envDynawo.sh:1382]: I don't get this line? Is this to remove folders that still exist in local but were deleted in git?

**Review by rosiereflo — APPROVED** [review]

**Comment by rosiereflo** [comment]
Could you please add this on the private side too?

File: util/envDynawo.sh (hunk 1 above)
Comment (gautierbureau) [reply util/envDynawo.sh:1382]: yes

**Comment by gautierbureau** [comment]
> Could you please add this on the private side too?

it is planned

**Pull request merged**